README.md

```diff
@@ -1 +1,211 @@
-# Budget-App
\ No newline at end of file
+# Budget Tracker App
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
```

Comment:
- [ ] [REQUIRED] Kindly uncomment this line so that a user can click it and get redirected to the LIVE DEMO section.